prody2/__init__.py

```diff
@@ -87,28 +87,43 @@ def addProDyPackage(cls, env, version, default=False):
         installCmd = [cls.getCondaActivationCmd()]
 
         # Activate the new environment
-        installCmd.append('conda activate %s;' % ENV_NAME)
+        installCmd.append('conda activate %s; pip install threadpoolctl;' % ENV_NAME)
```

Comment:
it is better to add this as a requirement to the plugin in the setup. Also, it is better to freeze the version (sometimes they update the version and add a new requirement, which can create library conflicts with other scipion packages)
You may add to the setup as `install_requires=['threadpoolctl==3.1.0'], or create requirements.txt file where all the requirements can be added (it gives better visibility to the requirements as well)
https://github.com/scipion-em/scipion-em-continuousflex/blob/master/setup.py#L149